themes/classic/templates/catalog/_partials/product-add-to-cart.tpl

```diff
@@ -53,6 +53,8 @@
             {l s='Add to cart' d='Shop.Theme.Actions'}
           </button>
         </div>
+
+        {hook h='displayProductActions' product=$product}
```

Comment:
maybe `displayProductCartActions` or even `displayProductAddToCartActions `

Reply:
Yes, I see your point I'm just afraid that `displayProductActions` is generic and you don't picture where this hook will be placed on the page And as I suggested `productCart` to match the template which contains it
But let's go for `displayProductActions`

Reply:
or  `displayProductAddToCartActions`